libs/blocks/section-metadata/section-metadata.js

```diff
@@ -12,11 +12,38 @@ function handleBackground(div, section) {
   }
 }
 
-function handleStyle(div, section) {
-  const value = div.textContent.toLowerCase();
-  const styles = value.split(', ').map((style) => style.replaceAll(' ', '-'));
+function handleStyle(div, section, customs = []) {
+  const value = div.textContent?.toLowerCase();
+  let styles = value.split(', ').map((style) => style.replaceAll(' ', '-')).filter(i => i !== '');
   if (section) {
-    section.classList.add(...styles);
+    if (customs.length) styles = [...styles, ...customs];
+    section.classList.add(...styles.filter(i => i !== ''));
```

Comment:
Why are you filtering twice? Seems redundant

Reply:
good catch, thanks!